procedures/katello/release.md.erb

```diff
@@ -26,12 +26,11 @@
   * [ ] Set version to `<%= version %>`
   * [ ] Set prerelease to `<%= extra %>`
 <% unless is_rc %>  * [ ] Reset release to `1`
-<% end %>  * [ ] In each package dir, remove the old tarball, run `spectool -g *.spec` and `git annex add *.tar.gz` or `git annex add *.gem`
-  * [ ] Edit `katello-repos/katello.repo` to set `gpgcheck=1`
-  * [ ] Commit with message "Release <%= full_version %>"
+<% end %>  * [ ] Fetch new source by running `obal source rubygem-katello`, then `git rm` the old source.
   * [ ] Add changelog entries: `obal changelog katello rubygem-katello katello-repos` --message "- Release <%= full_version %>"
+  * [ ] Commit with message "Release <%= full_version %>"
   * [ ] Submit a pull request
 * [ ] Merge packaging PR once job is green
 * [ ] Release all packages: `obal release katello katello-repos rubygem-katello`
```

Comment:
Isn't this handled upon PR merge now?

Reply:
No, we have no release jobs for stable branches. We did mention that we want it during our team meetup but nobody has gotten around to it yet. All cherry picks to stable branches require a manual release now :(

Reply:
:( kk